src/javascript/esutil.ts

```diff
@@ -157,12 +165,24 @@ export function toScannedPolymerProperty(
   let description =
       jsdoc.removeLeadingAsterisks(getAttachedComment(node) || '').trim();
 
+  const name = objectKeyToString(node.key);
+  if (name == null) {
```

Comment:
Here, I think you're totally right, `!name` is better, as an empty property name doesn't make sense as a polymer property.

In general I think we should keep `foo == null` in our repertoire, as it only catches `null` and `undefined` whereas `!foo` also catches `false`, `0`, `''`, `NaN`, (and in the browser, `document.all` because lol), and many times you do want to distinguish between nullish and falsy.